sdk/rust-examples/minigrep/src/main.rs

```diff
@@ -0,0 +1,53 @@
+//! Mini Grep
+//!
+//! ## Context
+//!
+//! Searches for the argument inside the input file, and prints the result.
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing and
+//! copyright information.
+
+use anyhow::{self, Ok};
+use std::{env, fs};
+
+const INPUT_FILENAME: &'static str = "/input/random_search_text.txt";
+const OUTPUT_FILENAME: &'static str = "/output/search_results.txt";
+
+fn main() -> anyhow::Result<()> {
+    let args: Vec<String> = env::args().collect();
+    let query: String = args.get(0).unwrap().to_string(); // Argument: To be searched for
```

Comment:
What if there is no argument passed?  What does this `unwrap()` do?

Reply:
I'll add a check for this!